.rubocop.yml

```
@@ -0,0 +1,23 @@
+AllCops:
+  NewCops: enable
+
+Style/StringLiterals:
+  Description: 'Checks if uses of quotes match the configured preference.'
+  StyleGuide: '#consistent-string-literals'
+  Enabled: true
+  VersionAdded: '0.9'
+  VersionChanged: '0.36'
+  EnforcedStyle: double_quotes
+  SupportedStyles:
+    - single_quotes
+    - double_quotes
+  ConsistentQuotesInMultiline: false
+
+Style/GlobalVars:
+  Description: 'Do not introduce global variables.'
+  StyleGuide: '#instance-vars'
+  Reference: 'https://www.zenspider.com/ruby/quickref.html'
+  Enabled: true
+  VersionAdded: '0.13'
+  # Built-in global variables are allowed by default.
+  AllowedVariables: [$window]
```

I don't have much experience with Rubocop, but I guess we only need these configuration lines?
```suggestion
Style/StringLiterals:
  EnforcedStyle: double_quotes

Style/GlobalVars:
  AllowedVariables: [$window]
```

---

That's correct, thanks for pointing that out!, however we need the bit of 
```yml
AllCops:
  NewCops:
```
to be enabled or disabled since RuboCop will give a warning otherwise, in this I'd prefer for it to be enabled, so the suggested solution can be this:
```yml
AllCops:
  NewCops: enable

Style/StringLiterals:
  EnforcedStyle: double_quotes

Style/GlobalVars:
  AllowedVariables: [$window]
```